src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.Calculator.UnitTest/NumberTranslatorTests.cs

```diff
@@ -132,12 +132,26 @@ public void Translate_RemoveNumberGroupSeparator_WhenCalled(string decimalSepara
         }
 
         [DataTestMethod]
-        [DataRow("12,0004", "12.0004")]
-        [DataRow("0xF000", "0xF000")]
-        public void Translate_NoRemovalOfLeadingZeroesOnEdgeCases(string input, string expectedResult)
+        [DataRow("de-DE", "12,0004", "12.0004")]
+        [DataRow("de-DE", "0xF000", "0xF000")]
+        [DataRow("de-DE", "0", "0")]
+        [DataRow("de-DE", "00", "0")]
+        [DataRow("de-DE", "12.0004", "120004")] // . is the group separator in de-DE
```

Comment:
Thousands-groups come in sets of three. Here are 4 digits together after one separator.
And (I think) would it be nice to include a larger number, meaning multiple separators. That, as both an integer and with decimals would be interesting.
Like, say, 3.141.592.653,5